dockerfiles/hpcc/templates/eclagent.yaml

```diff
@@ -12,8 +12,22 @@ spec:
       labels:
         run: eclagent
     spec:
+      volumes:
+      - name: dllserver-pv-storage
```

Comment:
I imagine thor and roxie require similar treatment

Reply:
Yes, they will. Though Roxie may need some additional work as it uses a customized dllserver to handle disconnections from Dali